tm2/pkg/amino/benchmark_test.go

```diff
@@ -6,14 +6,19 @@ import (
 	"runtime/debug"
 	"testing"
 
-	"github.com/gnolang/gno/tm2/pkg/amino"
-	"github.com/gnolang/gno/tm2/pkg/amino/tests"
 	fuzz "github.com/google/gofuzz"
```

Comment:
Just curious, why do we use the gofuzz package, since Go supports fuzzing out of the box as of 1.18? @moul

Reply:
I think because the code is older than the introduction of Fuzzing in Go